GitSearch/Models/PaddedText.swift

```diff
@@ -11,10 +11,11 @@ import UIKit
 class PaddedText: UITextField {
```

Comment:
It make sense to add `UIEdgeInsets` property with default value, so that everybody can tailor this subclass for their needs.

Reply:
done